deploy/docs/additional_prometheus_configuration.md

```diff
@@ -36,7 +36,7 @@ kube-prometheus-stack:  # For values.yaml
       # ...
       prometheusSpec:
         # ...
-        remoteWrite:
+        additionalRemoteWrite:
           # ...
```

Comment:
no need for `# ...` it was indication that there was something to copy

also, we should search documentation for all `remoteWrite` mentions, as I'm not sure but we probably mention in multiple places that you need to copy it from our `values.yaml` which is no longer true

Reply:
ok got it, will look for other references to remoteWrite. Also, I was able to send some metrics from the k8s-collector to a sumo instance. Will try adding additional prometheus metrics.

Reply:
get this error when I try to send Prometheus metrics to Sumo (long) via Fluentd.
config error file="/fluentd/etc/fluent.conf" error_class=Fluent::ConfigError error="Invalid SumoLogic endpoint url: "

it's not clear where in endpoint url gets configured in fluentd, configured the following in values.yaml
endpoint: "https://long-api......."

Reply:
Also @sumo-drosiek there doesn't seem to be any other mention of copying remoteWrite from values.yaml in the documentation

Reply:
Was able to send additional Prometheus metrics from a new metrics source. Also, removed copy references from the readme and added a note like below

NOTE: It is best practice to add custom configuration to a user-supplied values file and then use it like so (```helm install -f my_values.yaml -n sumologic```)